docs/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [3.0.0] - 2023-03-21
```

Comment:
Did you use `scripts/update_release.sh`? It should replace the Unreleased with the current date and version and prepare the `## Unreleased` for the future.

Reply:
I used the script but then manually edited the result which was not the correct procedure. I added the label back in https://github.com/RedHatProductSecurity/osidb/pull/170